scripts/download-bin.js

```diff
@@ -13,28 +13,24 @@ const dstDir = path.join(appRoot, 'build', 'probe-cli')
 
 const download = () => {
   ensureDirSync(dstDir)
-  const osarchlist = ['darwin_amd64', 'linux_amd64', 'windows_amd64']
-  for (let i = 0; i < osarchlist.length; i += 1) {
-    const osarch = osarchlist[i]
-    const tarball = `ooniprobe_${osarch}.tar.gz`
-    const tarballURL = `${baseURL}/${tarball}`
-    const sig = `${tarball}.asc`
-    const sigURL = `${tarballURL}.asc`
-    if (existsSync(`${dstDir}/${tarball}`) === false) {
-      console.log(`Downloading ${tarball}`)
-      execSync(`curl -#f -L -o ${dstDir}/${tarball} ${tarballURL}`)
-    }
-    if (existsSync(`${dstDir}/${sig}`) === false) {
-      console.log(`Downloading ${sig}`)
-      execSync(`curl -#f -L -o ${dstDir}/${sig} ${sigURL}`)
-    }
-    try {
-      execSync(`gpg --quiet --verify ${dstDir}/${sig} ${dstDir}/${tarball}`)
-    } catch (e) {
-      console.error(`Signature verification failure: ${e}`)
+  const osarchs = [
+    "darwin-amd64",
+    "linux-amd64",
+    "windows-amd64",
+  ]
+  for (let i = 0; i < osarchs.length; i += 1) {
+    const extension = (osarchs[i] === "windows-amd64") ? ".exe" : ""
```

Comment:
Maybe this could be better written as "if it contains windows then add extension". How do I do "if it contains" in JS?

Reply:
I added a commit that does this. (+ a few other JS stuff and lint fixes).